docs/en/observability/quickstarts/monitor-hosts-with-otel.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[quickstart-monitor-hosts-with-otel]]
+= Quickstart: Monitor hosts with OpenTelemetry
+
+preview::[]
+
+In this quickstart guide, you'll learn how to collect logs and metrics using the Elastic Distribution of OpenTelemetry (EDOT) Collector.
+You'll also learn how to get value out of your observability data after collecting it.
```

Comment:
Can we be more specific about what users will be able to do, "get value" this sounds too marketing abstract for a quickstart